tests/test_admin.py

```diff
@@ -10,21 +11,23 @@
 
 from cms.test_utils.testcases import CMSTestCase
 
-from djangocms_versioning.constants import DRAFT, PUBLISHED, UNPUBLISHED
-from djangocms_versioning.exceptions import ConditionFailed
-from djangocms_versioning.helpers import version_list_url
-
+from bs4 import BeautifulSoup
 from djangocms_navigation.admin import (
     MenuContentAdmin,
     MenuItemAdmin,
     MenuItemChangeList,
 )
 from djangocms_navigation.models import Menu, MenuContent, MenuItem
 from djangocms_navigation.test_utils import factories
+from djangocms_versioning.constants import DRAFT, PUBLISHED, UNPUBLISHED
+from djangocms_versioning.exceptions import ConditionFailed
+from djangocms_versioning.helpers import version_list_url
 
 from .utils import UsefulAssertsMixin, disable_versioning_for_navigation
 
 
+parse_html = partial(BeautifulSoup, features="lxml")
```

Comment:
I'm not a fan of BS, especially when it's used with the lxml parser. If it's lxml that you want to use I would advise using lxml.